src/Tools/GenerateProps/targetgroups.props

```diff
@@ -184,5 +184,8 @@
       <!-- this targetgroup is not considered compatible with any other targetgroup so it should only
            build when specified directly or BuildAllConfigurations is set to True.  -->
     </TargetGroups>
+    <TargetGroups Include="mono">
+      <NuGetTargetMoniker>.NETFramework,Version=v4.6.3</NuGetTargetMoniker>
```

Comment:
That's what I plan to do, add all the nested dependencies for a specific assembly. For example, for `System.Collections` that would be: `System.Diagnostics.Debug`, `System.Runtime` and `System.Private.Uri` (even if `System.Collections` doesn't depend on `System.Private.Uri` directly).